.github/workflows/build.yml

```diff
@@ -2,7 +2,7 @@
 name: Lint and test
 on:
   pull_request:
-    branches: [main, privatization]
+    branches: [main, release]
```

Comment:
Turns out it is very hard to release at the moment, as to merge a branch into `release` you must have the jobs in this workflow running. However, they did not trigger on this branch, so you are stuck.